packages/messaging-system/config/api-gateway.js

```diff
@@ -0,0 +1,12 @@
+// External dependencies
+const dotenv = require('dotenv');
+
+// Internal dependencies
+const path = require('./environment');
+
+dotenv.config({ path });
+
+module.exports = {
+  domain: process.env.DOMAIN_API_GATEWAY,
+  port: process.env.PORT_API_GATEWAY,
```

Comment:
Since this is going to used in the client-side you need to make some extra steps to expose this variables... check this documentation of NextJS: https://nextjs.org/docs/basic-features/environment-variables#exposing-environment-variables-to-the-browser